packages/spanner/src/database.js

```diff
@@ -577,6 +593,25 @@ Database.prototype.run = function(query, options, callback) {
  *   .on('end', function() {});
  *
  * //-
+ * // If you need to enforce a specific param type, a types map can be provided.
+ * // This is typically useful if your param value can be null.
```

Comment:
For those cases, we have the custom type methods like `spanner.int(8)`. If a user provided the values that was, as opposed to e.g. `8`, would that solve those cases?